elastic/kibana
# move esql editor to layer_panel.tsx

## Summary

moves esql editor to layer_panel.tsx as preparation to enable esql editing on each layer.

how to test this:
- create esql visualization in discover and put it on a dashboard
- edit visualization on the dashboard (esql query etc)
- everything should work exactly as before
